extensions/emmet/src/emmetCommon.ts

```diff
@@ -145,6 +146,14 @@ export function activateEmmetExtension(context: vscode.ExtensionContext) {
 			updateEmmetExtensionsPath(true);
 		}
 	}));
+
+	context.subscriptions.push(vscode.workspace.onDidOpenTextDocument((e) => {
+		addFileToParseCache(toLSTextDocument(e));
```

Comment:
I found `addFileToParseCache` confusing. Looking at the implementation, it set and entry with undefined to the cache. Is that necessary? it's done for all URI, regardless if html or not.

Reply:
Right, I should make it only save markup files like HTML or XML